.env.example

```diff
@@ -7,6 +7,7 @@ REACT_APP_SCOUT_URL=
 
 REACT_APP_API_HOST=
 REACT_APP_GOOGLE_CLIENT_KEY=
+REACT_APP_GOOGLE_STATIC_MAPS_API_KEY=
```

Comment:
I uhmed and ahhed for ages this is the safest way to do this since this key is exposed piblically and I wanted to keep it separate and it has usage limits as well